src/frontend/components/property-type/reference/edit.tsx

```diff
@@ -13,67 +13,78 @@ type SelectRecordEnhanced = SelectRecord & {
   record: RecordJSON;
 }
 
-class Edit extends React.Component<CombinedProps> {
-  constructor(props: CombinedProps) {
-    super(props)
-    this.loadOptions = this.loadOptions.bind(this)
-    this.handleChange = this.handleChange.bind(this)
-  }
+const Edit: FC<CombinedProps> = (props) => {
+  const { onChange, property, record, theme } = props
 
-  handleChange(selected: SelectRecordEnhanced): void {
-    const { onChange, property } = this.props
+  const handleChange = (selected: SelectRecordEnhanced): void => {
     if (selected) {
       onChange(property.name, selected.value, selected.record)
     } else {
       onChange(property.name, '')
     }
   }
 
-  async loadOptions(inputValue: string): Promise<Array<SelectRecordEnhanced>> {
-    const { property } = this.props
+  const loadOptions = async (inputValue: string): Promise<SelectRecordEnhanced[]> => {
     const api = new ApiClient()
 
     const records = await api.searchRecords({
       resourceId: property.reference as string,
       query: inputValue,
     })
-    return records.map((record: RecordJSON) => ({
-      value: record.id,
-      label: record.title,
-      record,
+    return records.map((r: RecordJSON) => ({
```

Comment:
once I received a comment in CR (say hallo to @plorencrstit) regarding not using shortcuts like `r`, `m`, and I am trying not to do this anymore. So hmmm - could you please change it to `record` :)

Reply:
love it!

Reply:
`record` is already there though. @plorencrstit can I have a copy of that comment? I'd like to know your reasoning on that.

I'll change it to `optionRecord`.

Reply:
It is about readability.  Variables should tells us what are they. `r` says nothing. Maybe in this situation ot is obvious for you that you mean `record` but there could be many other situation when it is not. Then, it is just a goos practice to avoid variables as `e` instead of `error`. Not much to effort, but the effects could be great.